src/base/bittorrent/cachestatus.cpp

```diff
@@ -50,7 +50,11 @@ qreal CacheStatus::readRatio() const
 
 int CacheStatus::jobQueueLength() const
 {
+#if LIBTORRENT_VERSION_NUM < 10100
     return m_nativeStatus.job_queue_length;
+#else
+    return m_nativeStatus.queued_jobs;
```

Comment:
yeah, the code you have should work (as long as deprecated functions in libtorrent are still enabled, which they are by default)